app/src/app/components/contents/explorer/Table.tsx

```diff
@@ -10,14 +10,22 @@ const headers: Header[] = [
 ];
 
 export default function Table({data}: PacketTableProps) {
-    const [ascending, setAscending] = useState(true);
+    const [ascendingName, setAscendingName] = useState(false);
+    const [ascendingStatus, setAscendingStatus] = useState(false);
```

Comment:
I really meant doing a more general approach here - I wouldn't be surprised if more sortable columns come along, and I don't think there's much value in retaining `ascending` values for columns which aren't the current sort column. I think you'd always expect the column to sort ascending when you switch from one sort column to another.

Something like the following..?

```suggestion
    const [ascending, setAscending] = useState(true);
    const [ascendingCol, setAscendingCol] = useState("");
```